package.json

```diff
@@ -9,23 +9,31 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@suankularb-components/css": "^2.7.3",
-    "@suankularb-components/react": "^2.6.1",
+    "@fontsource/inter": "^4.5.11",
+    "@fontsource/noto-sans-thai": "^4.5.9",
+    "@fontsource/sarabun": "^4.5.8",
+    "@fontsource/sora": "^4.5.8",
+    "@suankularb-components/css": "^2.7.4",
+    "@suankularb-components/react": "^2.6.2",
     "@supabase/supabase-js": "^1.35.2",
     "csv": "^6.0.5",
     "csv-parse": "^5.0.4",
     "date-fns": "^2.28.0",
     "framer-motion": "^6.2.8",
     "next": "^12.1.1",
     "next-i18next": "^10.5.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-markdown": "^8.0.2",
+    "react": "^17.0.2",
```

Comment:
I know we already talked but I’m gonna put this here for archival’s sake.
Because React SK Components just doesn't work with React 18.2.0 for some reason. Not good.